cypress/integration/js_center_spider.spec.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+describe(('Tests'), () => {
```

Comment:
Not "Tests", You should write here about what you describe in this block. Ex:
`Spider app` OR `Spider`, OR `Spider image` etc

Unneeded parenthesis
```suggestion
describe('Tests', () => {
```

Reply:
перейменував describe та it, видалив use strict